CHANGELOG.md

```diff
@@ -4,6 +4,11 @@ All notable changes to this project will be documented in this file
 automatically by Versionist. DO NOT EDIT THIS FILE MANUALLY!
 This project adheres to [Semantic Versioning](http://semver.org/).
 
+## v1.1.3 - 2018-04-11
+
+* Chore(package): Use npm instead of yarn #128 [Jonas Hermsmeier]
+* Fix: Replace Gitter link with forums link #128 [Andrew Scheller]
```

Comment:
That's weird - my fix was actually in #125

Reply:
Yeah, but versionbot can't merge external contributions, so that one got batched

Reply:
ping @hedss :wink: